tests/extendrtests/src/rust/src/lib.rs

```diff
@@ -46,6 +46,22 @@ fn char_scalar(x: String) -> String { x }
 #[extendr]
 fn char_vec(x: Vec<String>) -> Vec<String> {x}
 
+// Weird behavior of parameter descriptions:
+// first passes tests as is, second -- only in backqutoes.
+/// Test whether `_arg` parameters are treated correctly in R
+/// @param _x an integer scalar, ignored
+/// @param `_y` an integer scalar, ignored
+/// @export
+#[extendr]
+fn special_param_names(_x : i32, _y : i32) {}
```

Comment:
I suggest you do some non-commutative computation on `_x` and `_y` (e.g., `_x - _y`) and return, so you can test that the parameters are actually applied correctly in your R code.